services/ima/migrations/20240202143333_rename_uan_column_to_cepr.js

```diff
@@ -0,0 +1,12 @@
+
+exports.up = function(knex) {
+  return knex.schema.table('saved_applications', table => {
+    table.renameColumn('uan', 'CEPR');
```

Comment:
If sticking with lowercase, 'CEPR' should be lowercase too.

Reply:
Changed to lowercase too.